pkg/provider/aws/resources/vpc.go

```diff
@@ -86,15 +92,25 @@ func CreateNetwork(appName string, dag *core.ResourceGraph) *Vpc {
 	CreateInterfaceVpcEndpoint("sns", vpc, region, dag)
 	CreateInterfaceVpcEndpoint("secretsmanager", vpc, region, dag)
 
-	CreatePrivateSubnet(appName, "private1", vpc, "10.0.0.0/18", dag)
-	CreatePrivateSubnet(appName, "private2", vpc, "10.0.64.0/18", dag)
-	CreatePublicSubnet("public1", vpc, "10.0.128.0/18", dag)
-	CreatePublicSubnet("public2", vpc, "10.0.192.0/18", dag)
-
 	return vpc
 }
 
+func GetVpc(cfg *config.Application, dag *core.ResourceGraph) *Vpc {
+	for _, r := range dag.ListResources() {
+		if vpc, ok := r.(*Vpc); ok {
+			return vpc
+		}
+	}
+	return CreateNetwork(cfg.AppName, dag)
+}
+
+func GetSubnets(cfg *config.Application, dag *core.ResourceGraph) (sns []*Subnet) {
+	vpc := GetVpc(cfg, dag)
+	return vpc.GetVpcSubnets(dag)
+}
```

Comment:
i believe we have a GetVPCSbnets already which i think is more applicable because if we ever support multiple VPCs this wont work.